app/controllers/name_controller/create_and_edit_name.rb

```diff
@@ -253,19 +284,28 @@ def name_unlocked?
   end
 
   def minor_change?
+    return false if icn_id_changed?
+    return true if just_adding_author?
+
     old_name = @name.real_search_name
     new_name = @parse.real_search_name
     new_name.percent_match(old_name) > 0.9
   end
 
+  def icn_id_changed?
+    params[:name][:icn_id] != @name.icn_id
+  end
```

Comment:
I would call changing the ICN id a nontrivial change.  That could be very damaging for any agents that use the ICN id and expect it to be accurate.  I would only consider adding an ICN id where there was none before "minor".

Reply:
Agree. See above.